modules/pytket-quantinuum/pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -167,19 +183,15 @@ def _available_devices(
         return jr  # type: ignore
 
     @classmethod
-    def available_devices(cls, **kwargs: Any) -> List[BackendInfo]:
+    def available_devices(
```

Comment:
Not sure whether mypy would be happy with this signature against the `Backend.available_devices`